src/restconfig/src/test/java/org/geoserver/rest/catalog/FeatureTypeControllerTest.java

```diff
@@ -634,7 +634,7 @@ public void testCreateFeatureType() throws Exception {
                         + "  <attributes>\n"
                         + "    <attribute>\n"
                         + "      <name>the_geom</name>\n"
-                        + "      <binding>com.vividsolutions.jts.geom.MultiPolygon</binding>\n"
+                        + "      <binding>org.locationtech.jts.geom.MultiPolygon</binding>\n"
```

Comment:
It looks like the REST API describes attributes interns of their JTS class. While this does not amount to a REST API change, it is a change that is visible through the REST API and I wonder if downstream projects like gsconfig will be affected.

Checking now...

* https://github.com/boundlessgeo/gsconfig - no references
* https://github.com/geosolutions-it/geoserver-manager - one reference in a javadoc